components/ListView.js

```diff
@@ -357,7 +359,7 @@ ListView.propTypes = {
   getSectionId: PropTypes.func,
   hasFeaturedItem: PropTypes.bool,
   keyExtractor: PropTypes.func,
-  ListEmptyComponent: PropTypes.node,
+  ListEmptyComponent: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
```

Comment:
It can't be `PropTypes.node`? Just double checking, not sure myself.

Reply:
I can't remember last time it was node.
In Shoutem, it's usually a function or object - function for "plain" component, object if component is wrapped in connectStyle (if I remember right, last time I checked)